doc/source/features.rst

```diff
@@ -79,6 +79,18 @@ Command line usage will use the new names ::
     positional arguments:
       {friendly_func,func2}
 
+Nested Subcommands
+------------------
+
+By using dictionaries, the subcommands can also be nested.
+
+.. code-block:: python
+
+    defopt.run({'func1': func1, 'sub': [func2, func3]})
+
+The nested subcommands are accessed e.g. by ``test.py sub func2``.  The
+subcommands can be nested to an arbitrary level by using nested dictionaries.
```

Comment:
Also link to the runnable example (see `examples/foo.py` links that occur elsewhere in the page).

Reply:
Added it.